mne/minimum_norm/psf_ctf.py

```diff
@@ -241,14 +219,16 @@ def _get_matrix_from_inverse_operator(inverse_operator, forward, labels=None,
     invmat : ndarray
         Inverse matrix associated with inverse operator and specified
         parameters.
-    label_singvals : list of ndarray
-        Singular values of svd for sub-inverses.
-        Provides information about how well labels are represented by chosen
-        components. Explained variances within sub-inverses are shown in
-        screen output.
+    label_singvals : list of dict
+        Singular values of SVD for sub-matrices of inverse operator
+        (only if mode='svd').
+        Diffent list entries per label. Since SVD is applied separately for
+        channel types, dictionaries may contain keys 'mag', 'grad' or 'eeg'.
     """
     mode = mode.lower()
 
+    # apply_inverse cannot produce 3 separate orientations
+    # therefore 'force_fixed=True' is required
```

Comment:
This is true (for surface source spaces anyway) only until #3842 lands. Maybe it's worth waiting for that?